src/main/java/codeu/model/store/basic/MentionStore.java

```diff
@@ -67,6 +73,43 @@ public Mention getMentionById(UUID id) {
     }
     return null;
   }
+  
+  /**
+   * Access the current list of mentions by the given user UUID.
+   *
+   * @return null if the user has not mentioned another user.
+   */
+  public List<Mention> getMentionsByUserId(UUID id) {
+	  List<Mention> result = new ArrayList<>();
+	  for (Mention mention : mentions) {
+		  if (mention.getUserWhoDidTheMentioning().equals(id)) {
+			  result.add(mention);
+		  }
+	  }
+	  if (result.size() > 0) {
+		  return result;
+	  }
+	  return null;	  
+  }
+  
+  /**
+   * Access the current list of mentions for the given user UUID.
+   *
+   * @return null if the user has been mentioned by another user.
+   */
+  public List<Mention> getMentionsForUserId(UUID id) {
+	  List<Mention> result = new ArrayList<>();
+	  for (Mention mention : mentions) {
+		  if (mention.getUserWhoWasMentioned().equals(id)) {
+			  result.add(mention);
+		  }
+	  }
+	  if (result.size() > 0) {
```

Comment:
similar here -- just return an empty list.  Java is annoying with nulls because it throws exceptions and kills the program, so it's usually much nicer to return an empty collection. If you need to find out if it returned results, just check .size() == 0 on the resulting value.